src/main/scala/mesosphere/marathon/core/plugin/impl/PluginManagerImpl.scala

```diff
@@ -94,7 +94,7 @@ object PluginManagerImpl extends StrictLogging {
 
   def parse(fileName: String): PluginDefinitions = {
     val confJson: JsObject = Json.parse(FileUtils.readFileToByteArray(new File(fileName).getCanonicalFile)).as[JsObject]
-    logger.info(s"Found plugin configuration: ${Json.prettyPrint(confJson)}")
+    logger.debug(s"Found plugin configuration: ${Json.prettyPrint(confJson)}")
```

Comment:
You should also change the log level of the message at line 44:

```logger.info(s"Configure the plugin with this configuration: ${definition.configuration}")```

Reply:
good catch, done